middleware/jwt/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "gotham_middleware_jwt"
+version = "0.3.0"
+authors = ["Nicholas Young <nyoung@uptime.ventures>",
+          "Colin Bankier <colinbankier@gmail.com>",
+          "Isaac Whitfield <iw@whitfin.io>",
+          "Judson Lester <nyarly@gmail.com>",
+          "Bradley Beddoes <bradleybeddoes@gmail.com>"]
+description = "JWT middleware for the Gotham web framework."
+repository = "https://github.com/gotham-rs/gotham"
+keywords = ["gotham-middleware", "jwt", "jsonwebtoken", "authentication"]
+homepage = "https://gotham.rs"
+readme = "README.md"
+license = "MIT/Apache-2.0"
+
+[dependencies]
+futures = "0.1.25"
```

Comment:
Can you pin these versions only to the minor? i.e. `futures = 0.1`?

Reply:
Done.